src/sandstorm/grain.capnp

```diff
@@ -633,7 +633,20 @@ interface SessionContext {
   #   saveLabel: A string petname to give this label. This will be displayed to the user as the name
   #          for this capability.
   #
-  # (eg. window.parent.postMessage({powerboxRequest: {rpcId: myRpcId, query: [{}]}}, "*")
+  # e.g.:
+  #   window.parent.postMessage({
+  #     powerboxRequest: {
+  #       rpcId: myRpcId,
+  #       query: [
+  #         {
+  #           tags: [
+  #             { id: "10145426045091321204" },
```

Comment:
I'm trying to follow the node-capnp encoding rules as closely as possible, here, so that one day this could become more strongly typed without breaking compatibility.  `id` here is really "a capnproto uint64", but because JS can't represent a 64-bit numeric type, we fake it with strings.

For some reason, I though that node-capnp only accepted base-10 strings as 64-bit numerics, but looking at `node-capnp/src/node-capnp/capnp.cc:1224` it looks like we actually accept anything that `strtoull(3)` accepts.  So I guess I can switch everything to hex ids after all!

In which case I should probably do so everywhere.